packages/next/src/server/future/route-modules/app-route/helpers/auto-implement-methods.ts

```diff
@@ -0,0 +1,82 @@
+import type { AppRouteHandlerFn, AppRouteHandlers } from '../module'
+
+import { HTTP_METHODS, type HTTP_METHOD } from '../../../../web/http'
+import { handleMethodNotAllowedResponse } from '../../helpers/response-handlers'
+
+const AUTOMATIC_ROUTE_METHODS = ['HEAD', 'OPTIONS'] as const
+
+export function autoImplementMethods(
```

Comment:
These methods in `packages/next/src/server/future/route-modules/app-route/helpers` are just the helper functions that used to live inside the old `app-route-route-handler.ts` file.